3_javascript/javascript1/week3/Item array removal/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Neat file organization in the folders! 
Impressive systematic approach. 
There is a caveat, though, when trying to operate through your hw and f.x. run it with bash:
https://apple.stackexchange.com/questions/14683/how-to-cd-to-a-directory-with-a-name-containing-spaces-in-bash
Worth considering updating the naming convention 🙏🏽

Reply:
Thanks! 
Regarding the naming, you are right, I had not thought about it, I will keep it in mind from now on :)